zipline/testing/core.py

```diff
@@ -492,20 +490,17 @@ def create_data_portal(env, tempdir, sim_params, sids, adjustment_reader=None):
         )
 
 
-def write_bcolz_minute_data(env, days, path, df_dict):
+def write_bcolz_minute_data(env, days, path, data):
```

Comment:
Confirming my understanding, main intent of this patch is to enable `data` here as any given iterable of `sid, frame`.

Looks good to me.